src/LiveComponent/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # CHANGELOG
 
+## 2.24.0
+
+-   Add uid support for hydration
```

Comment:
Since https://github.com/ramsey/uuid also exists, let's be more explicit:

```suggestion
-   Add support for [Symfony UID](https://symfony.com/doc/current/components/uid.html) hydration/dehydration
```